python-sdk/docs/astro/sql/operators/transform_sql.rst

```diff
@@ -0,0 +1,16 @@
+.. _transform_sql_operator:
+
+=======================
+transform_sql operator
+=======================
+
+When to use the ``transform_sql`` operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The ``transform_sql`` operator allows you to implement the **T** of an ELT system by running a SQL query from specified SQL string. Each step of the transform pipeline creates a new table from the ``SELECT`` statement and enables tasks to pass those tables as if they were native Python objects.
```

Comment:
It is worth explaining when the user should be using this compared to `transform_file` or `transform`.